proposals/new-project-donation-process.md

```diff
@@ -0,0 +1,69 @@
+# KEP-748: Project Donation to Kubeflow
+
+## Summary
+This KEP outlines how community members can donate their project to the Kubeflow community.
+
+## Motivation
+As Kubeflow has become a well established ecosystem and community, several
+projects may want to be donated to the Kubeflow platform to explicitly join our 
+community.
+
+### Goals
```

Comment:
Should one of the goals be to explain why it is valuable for Kubeflow community to accept new projects ?

Reply:
I added a sentence about this. 👍